lib/deafult_campaign_update.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```

Comment:
Filename is misspelled.

Reply:
Fixed in fd9a70f9c5cded399d1a84ae6c04271312717bd8